### PR TITLE
Adding WordPress demos

### PR DESCRIPTION
**Pull request by erikaheidi**
Adding 3 wordpress demos for upcoming "getting started with Wordpress" guide

### CODE REVIEW

File: php/wordpress/03-distroless/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM cgr.dev/chainguard/wordpress:latest-dev as builder
```
Comment (amouat) [inline php/wordpress/03-distroless/Dockerfile:1]: docker build warns that the "as" should be "AS"

File: php/wordpress/02-customizing/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM cgr.dev/chainguard/wordpress:latest-dev
+ARG UID=1000
+
+USER root
+RUN apk add shadow
```
Comment (amouat) [inline php/wordpress/02-customizing/Dockerfile:5]: Why is this needed? Should it be in the base image?

File: php/wordpress/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# WordPress Chainguard Image Demos
+
+This directory contains a set of demos that show how to use the ChainGuard WordPress Image.
+
+All demos use Docker Compose to set up a LEMP environment composed by three services: PHP-FPM*, MariaDB and Nginx. The environment variables are set in a `.env` file that is included in each demo and can be modified to suit your needs.
```
Comment (amouat) [inline php/wordpress/README.md:5]: Maybe link or explain LEMP

Comment (amouat) [reply php/wordpress/README.md:5]: Also "composed *of* three services"

**Review by amouat — APPROVED** [review]
LGTM! This pretty much all just worked. Some minor comments on Dockerfiles and the README

File: php/wordpress/02-customizing/Dockerfile (hunk 2 above)
Comment (erikaheidi) [reply php/wordpress/02-customizing/Dockerfile:5]: it's for the `useradd` command! without this user is pretty hard to make volumes work. I have considered adding this user to the -dev image only, but it was a bit complicated to achieve that only for the -dev variant.

Comment (amouat) [reply php/wordpress/02-customizing/Dockerfile:5]: Isn't `adduser` there already?

Comment (erikaheidi) [reply php/wordpress/02-customizing/Dockerfile:5]: ooooh gosh, I didn't know that :D updating it now
